src/validation/enums/DomainErrors.enum.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @enum {DomainErrorMessagesEnum} DomainErrorMessagesEnum - Definition of messages displayed in errors
+ *
+ * @arg {string} InvalidFieldError - Invalid field message error
+ * @arg {string} RequiredFieldError - Required field message error
+ */
+export enum DomainErrorMessagesEnum {
+    InvalidFieldError = "Invalid value",
```

Comment:
use 2 spaces and simple quotes in all file

Reply:
I fix this question @rubemfsv